airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/SnowflakeBeanFactory.kt

```diff
@@ -212,15 +212,14 @@ class SnowflakeBeanFactory {
         return if (dataChannelMedium == DataChannelMedium.STDIO) {
             MemoryAndParallelismConfig(
                 maxRecordsPerAgg = 10_000_000_000_000L,
-                maxEstBytesPerAgg = 350_000_000L,
+                maxEstBytesPerAgg = 100_000_000L,
             )
         } else {
             MemoryAndParallelismConfig(
                 maxRecordsPerAgg = 10_000_000_000_000L,
-                maxEstBytesPerAgg = 350_000_000L,
                 maxBufferedAggregates = 6,
+                maxEstBytesPerAgg = 100_000_000L,
```

Comment:
what's the rationale behind the estimate?

Reply:
I need to rename all of these

Reply:
This is jons branch where the aggregates are held in memory instead of being written to disk, so 350mb is way too big to fit 5-10 into memory. 100mb works. Might be able to go bigger.